Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/SeedingAlgorithm.hpp

```diff
@@ -100,8 +111,8 @@ class SeedingAlgorithm final : public IAlgorithm {
   WriteDataHandle<SimSeedContainer> m_outputSeeds{this, "OutputSeeds"};
 
   static inline bool itkFastTrackingCuts(float bottomRadius, float cotTheta) {
-    float RMin = 50.;
-    float CotThetaMax = 1.5;
+    static float RMin = 50.;
+    static float CotThetaMax = 1.5;
```

Comment:
potentially `static constexpr` outside the function?
```suggestion
    static float rMin = 50.;
    static float cotThetaMax = 1.5;
```